docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml

```
@@ -1238,19 +1546,48 @@ subquestion: |
   
   **Elapsed time: ${ str( tests_elapsed_time.delta ) }**[BR]
   (Updates about every 10 seconds depending on your server speed)
+
+  % if stopped_early:
+  
+  <div class="alert alert-info" markdown=1>The tests are stopping, honest. They really are doing their best. Here's yet another button. It won't do much, but it might help you feel better `(͡◵ Ĺ̯ ͡◵ )`</div>
+  <div id="frustration">
+  ${ action_button_html("javascript:console.log(`Aaaand it also does this. Can't pull one over on you! I should know better by now.`, $(`#frustration_result`)[0]); $(`#frustration_result`)[0].style.display=`block`; $(`#frustration .btn`)[0].textContent += ` Go!`; document.body.style.backgroundColor = `rgb(` + (Math.random() * 100 + 100) + `,` + (Math.random() * 100 + 100) + `,` + (Math.random() * 100 + 100) + `)`; if ( daCheckin ) { daCheckin(); }", label="Go!", color="danger", size="md""") }
+  </div>
+  <div id="frustration_result" style="display: none;">Ok, it does do this stuff. That's all, though.</div>
+  % endif
 action buttons:
   - label: Stop tests early
     action: stop_tests_early
     icon: window-close
     color: danger
+    show if: not stopped_early
+comment: |
+  Relevant Lenny faces
+  ( ◵  ͟ʖ ◵ )
+  乁( ⁰͡ Ĺ̯ ⁰͡ ) ㄏ
+  (⊙﹏⊙)
+  (͡• ͟ʖ ͡•)
+  (͡◵ Ĺ̯ ͡◵ ;)
+  ( ͡° _ʖ ͡°)
+  ( ͡°Ĺ̯ ͡° )
+  ( ᵒ̴̶ Ĺ̯ ᵒ̴̶ )
+  (ᵒ̴̶﹏ᵒ̴̶)
+  .·°՞(ᵒ̴̶ᯅᵒ̴̶)՞°·.
+  (o̴̶̷᷄⸝⸝·̭ o̴̶̷̥᷅⸝⸝)
+  (o̴̶̷᷄ ·̭ o̴̶̷̥᷅)
+  (ó﹏ò｡)
+  (ó_ò｡)
+  (ᵒ̴̶̷᷄﹏ᵒ̴̶̷᷅)
+  (◕_◕)
 ---
 event: stop_tests_early
 code: |
+  _debug("#### stop_tests_early event ####")
```

Perhaps add this type of log for stopping install early as well.

```suggestion
  _debug("Event: stop_tests_early")
```